### PR TITLE
Feature/damien/enhanced video results

### PR DESCRIPTION
**Pull request by idomo1**
## Proposed changes

Enhance the video results by proving explanations and linking out to Material Design resources.
I didn't do the same thing for images, since there isn't really much to be explained.
There are only three possible image defects:
- Null value
- Component occlusion
- Missing image
There isn't much to expand on...

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] DevOps related

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [project guidelines](https://github.com/Visual-Testing-for-Android-Apps/Project-guidelines) doc
- [x] I locally linted my code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

https://user-images.githubusercontent.com/53201250/130590371-b52e4e6c-60d8-45cc-8aeb-a54ecf97a5e6.mov

Screen Shot 2021-08-24 at 7 11 00 pm
Screen Shot 2021-08-24 at 7 11 21 pm

### CODE REVIEW

**Comment by idomo1** [comment]
I branched this off of the colour scheme picker branch, so wait for that one to be merged before reviewing this.

**Review by leanne-b — COMMENTED** [review]
**Desktop View:**
- I like the description boxes and the fonts they look nice
- It would be nice if the video with error analysing video could be aligned with the other normal videos, right now it looks out of alignment because the description box isn't the same size as the other videos. Please align the videos in a column with their left edges lining up, similarly for the description boxes. 
- It might be help to have two columns to help align things with the video on the left and the descriptions in the right column or somehow make all the description boxes the same size so when they're centred the vids+descrip are all in alignment. 
image

- I think the description style can be more consistent, they look differently spread in the image below. 
image
- The flow layout is weird at zoom <=90%, full screen
image
- If the colour scheme selector is aligned left, then everything below should be aligned left for consistency

**Mobile View:**
- On Mobile it can be a little bit confusing to differentiate which feedback is for which video, I thought the "missing scrim" was for the video below because it looked like a heading but it was actually for the video above. 

image

- I think it would be good to have a heading above each image or video in the font style similar to the "missing scrim". Or something visual to help differentiate between the files and the feedback that belongs to it. 
- Move the description above the video instead? 

**Tablet view:**
- images section of the report gives an empty feel because there is a lot of space on both sides and small short text
image

- Video section: if you keep the media centred, I think you need padding/margins on the left+right of description part for missing scrim so it doesn't spread out towards the edges so much
image

**Comment by mt-empty** [comment]
Yea images and videos are not consistent, becuase of the image result limitation, but I was thinking maybe we could isolate image from video results, that way its more consistent, e.g. rows spesific for images and rows specific for videos 

This is how it looks like on my desktop
image

File: src/components/reportPage/results-page.css [hunk 1]
```diff
@@ -82,5 +92,18 @@
...
+  .video-result-container {
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .video-result-explanation {
+    margin: 0 4rem;
+    border: 0.06rem solid grey;
+    padding: 3rem;
+    max-width: 30rem;
```
Comment (mt-empty) [inline src/components/reportPage/results-page.css:107]: Why is there two of `video-result-explanation` and `video-result-container` in this file?

**Review by mt-empty — CHANGES_REQUESTED** [review]
This is better than mine, I left some comments.

File: src/components/reportPage/ImageResult.jsx [hunk 2]
```diff
@@ -78,16 +80,24 @@ const ImageResult = ({ imageResult, colourScheme }) => {
...
+      {isError ? (
+        <div className="result">
+          <p>Error analysing image</p>
+        </div>
+      ) : (
+        <>
+          <div className="result">
+            <canvas ref={originalImageCanvasRef} className="original-image" />
+            <canvas ref={resultImageCanvasRef} className="image-heatmap" />
+          </div>
```
[the remark this answers is not in the document]
Comment (idomo1) [reply src/components/reportPage/ImageResult.jsx:93]: You've already done it on your branch I think so I'll leave it

**Review by Akshith-Patil — APPROVED** [review]
Looks good!

**Review by mt-empty — APPROVED** [review]

**Pull request merged**